introduction.md

```diff
@@ -88,30 +91,30 @@ The following changes and additions have been made to update the 2013 WCAG2ICT d
     * [Success Criterion 2.5.7 Dragging Movements](#dragging-movements)
     * [Success Criterion 2.5.8 Target Size (Minimum)](#target-size-minimum)
     * [Success Criterion 3.2.6 Consistent Help](#consistent-help)
     * [Success Criterion 3.3.7 Redundant Entry](#redundant-entry)
     * [Success Criterion 3.3.8 Accessible Authentication](#accessible-authentication-minimum)
-* Obsolete and Removed WCAG 2.2 Success Criteria that have errata for WCAG 2.0 and 2.1
+- Obsolete and Removed WCAG 2.2 Success Criteria that have errata for WCAG 2.0 and 2.1
     * [Success Criterion 4.1.1 Parsing](#parsing)
-* New terms from WCAG 2.1 and 2.2
+- New terms from WCAG 2.1 and 2.2
     * dragging movements, encloses, focus indicator, minimum bounding box, pointer input, process, single pointer, state, status message were added to [Glossary Items that Apply to All Technologies](#glossary-items-that-apply-to-all-technologies)
     * motion animation, region, and user inactivity were added to [Glossary Items Used only in AAA Success Criteria](#glossary-items-used-only-in-aaa-success-criteria)
-    * added to [Glossary Items with Specific Guidance](#glossary-items-with-specific-guidance): 
-        * [cognitive function test](#dfn-cognitive-function-test)
-        * [css pixel](#dfn-css-pixel)
-        * [down event](#dfn-down-event)
-        * [keyboard shortcut](#dfn-keyboard-shortcut)
-        * [perimeter](#dfn-perimeter)
-        * [style property](#dfn-style-property)
-        * [target](#dfn-target)
-        * [up event](#dfn-up-event)
-* Updated terms
+    * added to [Glossary Items with Specific Guidance](#glossary-items-with-specific-guidance):
+        - [cognitive function test](#dfn-cognitive-function-test)
+        - [css pixel](#dfn-css-pixel)
+        - [down event](#dfn-down-event)
+        - [keyboard shortcut](#dfn-keyboard-shortcut)
+        - [perimeter](#dfn-perimeter)
+        - [style property](#dfn-style-property)
+        - [target](#dfn-target)
+        - [up event](#dfn-up-event)
+- Updated terms
     * [set of web pages](#dfn-set-of-web-pages)
     * [set of non-web documents](#set-of-documents)
     * [set of software programs](#set-of-software-programs)
-* Updated sections
-    <div class=note">In this draft, most of the existing sections have undergone WCAG2ICT Task Force review and updates. Many sections required only minor editorial and link URL updates, such as the guidance for each WCAG 2.0 success criteria. Any sections that have not been fully updated have editor's notes to reflect their current status.</div>
+- Updated sections
+<div class=note">In <em>this</em> draft, most of the existing sections have undergone WCAG2ICT Task Force review and updates.  Many sections required only minor editorial and link URL updates, such as the guidance for each WCAG 2.0 success criteria.  Any sections that have not been fully updated have editor's notes to reflect their current status.</div>
```

Comment:
```suggestion
<div class=note">In <em>this</em> draft, most of the existing sections have undergone WCAG2ICT Task Force review and updates.  Many sections required only minor editorial and link URL updates, such as the guidance for each WCAG 2.0 success criteria.</div>
```

Reply:
We will be doing a sweep of editor's note changes before publication, but since this was being edited, I removed what we don't need. (see above suggested changes.)